packages/react-core/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@aws-amplify/ui-react-core",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "files": [
+    "dist",
+    "LICENSE"
+  ],
+  "scripts": {
+    "prebuild": "rimraf dist",
```

```suggestion
    "clean": "rimraf dist",
    "prebuild": "rimraf dist node_modules",
```
I'd add a `clean` here to be consistent with the other monorepo packages